lib-api/src/main/java/org/ergoplatform/appkit/BoxOperations.java

```diff
@@ -115,6 +116,21 @@ public BoxOperations withAttachment(@Nullable BoxAttachment attachment) {
         return this;
     }
 
+    public int getMaxInputBoxesToSelect() {
+        return maxInputBoxesToSelect;
+    }
+
+    /**
+     * @param maxInputBoxesToSelect if set greater than 0, {@link #loadTop()} will only select
+     *                              up to this number of input boxes to satisfy ERG and token amount
+     *                              needed and throws an {@link org.ergoplatform.appkit.InputBoxesSelectionException.InputBoxLimitExceededException}
+     *                              otherwise
```

Comment:
Please describe what the default value of 0 means.

Reply:
Done